includes/class-wc-abstract-google-analytics-js.php

```diff
@@ -230,6 +230,13 @@ public function get_formatted_product( WC_Product $product, $variation_id = 0, $
 			}
 		}
 
+		// Integration with Product Bundles.
+		// Get the minimum price, as `get_price` may return 0 if the product is a bundle and the price is potentially a range.
+		// Even a range containing a single value.
+		if ( class_exists( '\WC_Product_Bundle' ) && $product instanceof WC_Product_Bundle && is_callable( [ $product, 'get_bundle_price' ] ) ) {
```

Comment:
Had you considered simplifying this to:

```suggestion
		if ( $product->is_type( 'bundle' ) && is_callable( [ $product, 'get_bundle_price' ] ) ) {
```

Granted another extension may be registering it's own product type called `bundle` but in that scenario the class name may also match and as we have the second check to confirm the `get_bundle_price` method exists I think it would be suitable.